Article/Article.js

```diff
@@ -1,29 +1,33 @@
-// Because classes are not hoised you will need to start your code at the bottom of the page.  Look for the comment "START HERE"
+// Because classes are not hoised you will need to start your code at the bottom of the page.
+// Look for the comment "START HERE"
 
 class Article {
   constructor(domElement) {
     // assign this.domElement to the passed in domElement
-    this.domElement;
+    this.domElement = domElement
     // create a reference to the ".expandButton" class. 
-    this.expandButton;
+    this.expandButton = this.domElement.querySelector('.expandButton')
     // Using your expandButton reference, update the text on your expandButton to say "expand"
-    
+    this.expandButton.innerText = "expand"
     // Set a click handler on the expandButton reference, calling the expandArticle method.
-    
+    this.expandButton.addEventListener('click', this.expandArticle.bind(this))
```

Comment:
try using arrow functions to bind your actions so that you can leave the world of this binding behind. Keeping track of when to bind this is nice to know but can be a pain for sure.